doc_src/license.hdr

```diff
@@ -9,7 +9,9 @@
 
 `fish` Copyright © 2005-2009 Axel Liljencrantz. `fish` is released under the GNU General Public License, version 2.
 
-`fish` includes other code licensed under the GNU General Public License, version 2, including GNU `printf`:
+`fish` includes other code licensed under the GNU General Public License, version 2, including GNU `printf`.
+
+`fish` includes code licensed under the MIT License, such as the MuParser math expression evaluation library.
```

Comment:
Okay, but the preceding paragraph seems to say the same thing albeit without the explicit "such as":

> `fish` includes other code licensed under the GNU General Public License, version 2, including GNU `printf`.

A normal reading of that sentence is that our `printf` implementation is only one of possibly several pieces of code licensed under GPL v2. But what do I know. IANAL :smile: So I'll drop the "such as".